src/renderer/app.tsx

```diff
@@ -1,32 +1,123 @@
 import React, { FC, Fragment, useEffect, useState } from 'react';
+import { Accordion, Button, ButtonGroup, Card, ToggleButton } from 'react-bootstrap';
 
 interface Tweet {
   text: string;
   videoPath?: string;
+  id: string;
 }
 
+type ThemeSelectorProps = {
+  radioValue: string;
+  setRadioValue: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const ThemeSelector = ({ radioValue, setRadioValue }: ThemeSelectorProps) => {
+  const radios = [
+    { name: 'Dark', value: 'dark' },
+    { name: 'Light', value: 'light' },
+  ];
+  return (
+    <ButtonGroup toggle>
+      {radios.map((radio, idx) => (
+        <ToggleButton
+          key={idx}
+          type="radio"
+          variant="secondary"
+          name="radio"
+          value={radio.value}
+          checked={radioValue === radio.value}
+          onChange={(e) => setRadioValue(e.currentTarget.value)}>
+          {radio.name}
+        </ToggleButton>
+      ))}
+    </ButtonGroup>
+  );
+};
+
+type TweetContProps = {
+  tweets: Tweet[];
+  theme: string;
+};
+
+const TweetContainer = ({ tweets, theme }: TweetContProps) => {
+  const [activeKeyState, setActiveKey] = useState<string>('');
+
+  const handleActiveKeyChange = (activeKey: string) => {
+    if (activeKeyState === activeKey) {
+      //allows us to close expanded item by clicking its toggle while open
+      activeKey = '';
+    }
+    setActiveKey(activeKey);
+  };
+
+  return (
+    <Accordion activeKey={activeKeyState} className={theme}>
+      {tweets.map((tweet) => (
+        <Card key={tweet.id}>
+          <Card.Header>
+            <Accordion.Toggle
+              as={Button}
+              variant="link"
+              eventKey={tweet.id}
+              onClick={() => handleActiveKeyChange(tweet.id)}>
+              {tweet.text.split(' ').slice(0, 15).join(' ')}
+            </Accordion.Toggle>
+          </Card.Header>
+          <Accordion.Collapse eventKey={tweet.id}>
+            <div>
+              <Card.Body>{tweet.text.split(' ').slice(15).join(' ')}</Card.Body>
+              {tweet.videoPath ? <video controls playsInline src={tweet.videoPath} /> : <div></div>}
```

Comment:
Maybe you can simply do
```
tweet.videoPath && <video controls playsInline src={tweet.videoPath} />
```
Because `null` and `undefined` will not be rendered by React